pkg/combustion/elemental.go

```diff
@@ -24,10 +24,9 @@ const (
 var elementalScript string
 
 func configureElemental(ctx *image.Context) ([]string, error) {
-
 	if !isComponentConfigured(ctx, elementalConfigDir) {
 		log.AuditComponentSkipped(elementalComponentName)
-		zap.S().Info("Skipping elemental registration component, configuration is not provided")
+		zap.S().Info("skipping elemental registration component, configuration is not provided")
```

Comment:
I'm pretty sure 95%+ of the log messages are capitalised. Do we want to use lower cased ones instead?

Reply:
I may have been confusing myself with error messages. I'm happy to revert.